src/Aspire.Hosting/DistributedApplication.cs

```diff
@@ -32,13 +64,75 @@ public DistributedApplication(IHost host)
     /// Creates a new instance of the <see cref="IDistributedApplicationBuilder"/> interface.
     /// </summary>
     /// <returns>A new instance of the <see cref="IDistributedApplicationBuilder"/> interface.</returns>
+    /// <remarks>
+    /// This overload of the <see cref="CreateBuilder()"/> method should only be
+    /// used when the AppHost is not intended to be used wtih a deployment tool. Because no arguemnts are
+    /// passed to the <see cref="CreateBuilder()"/> method the AppHost has no
+    /// way to be put into publish mode. Refer to <see cref="CreateBuilder(string[])"/> or  <see cref="CreateBuilder(DistributedApplicationOptions)"/>
+    /// when more control is needed over the behavior of the distributed application at runtime.
+    /// </remarks>
+    /// <example>
+    /// The following example shows creating a Postgres server resource with a database and referencing that
+    /// database in a .NET project.
+    /// <code>
+    /// var builder = DistributedApplication.CreateBuilder();
+    /// var inventoryDatabase = builder.AddPostgres("mypostgres").AddDatabase("inventory");
+    /// builder.AddProject&lt;Projects.InventoryService&gt;()
+    ///        .WithReference(inventoryDatabase);
+    ///
+    /// builder.Build().Run();
+    /// </code>
+    /// </example>
     public static IDistributedApplicationBuilder CreateBuilder() => CreateBuilder([]);
 
     /// <summary>
     /// Creates a new instance of <see cref="IDistributedApplicationBuilder"/> with the specified command-line arguments.
     /// </summary>
     /// <param name="args">The command-line arguments to use when building the distributed application.</param>
     /// <returns>A new instance of <see cref="IDistributedApplicationBuilder"/>.</returns>
+    /// <remarks>
+    /// <para>
+    /// The <see cref="DistributedApplication.CreateBuilder(string[])"/> method is the most common way to
+    /// create an instance of the <see cref="IDistributedApplicationBuilder"/> interface. Typically this
+    /// method will be called as a top-level statement in the application's entry-point.
+    /// </para>
+    /// <para>
+    /// Note that the <paramref name="args"/> parameter is a string is essential in allowing the application
```

Comment:
```suggestion
    /// Note that the <paramref name="args"/> parameter is a string which is essential in allowing the application
```

Also use a tag for `string`?

Reply:
For `string` or `null` or any C# language keyword, we should use it wrapped in the following `<see langword="string" />`. For example,, see these examples: https://grep.app/search?q=langword&filter[repo][0]=dotnet/runtime